Source/NSDistributedNotificationCenter.m

```diff
@@ -734,18 +734,39 @@ - (void) _connect
 		@"--auto",
 		nil];
 	    }
-	  [NSTask launchedTaskWithLaunchPath: cmd arguments: args];
-
+#if defined(__MINGW32__)
+          NSTask *task = AUTORELEASE([NSTask new]);
+          [task setStandardError:[NSFileHandle fileHandleForWritingAtPath:@"NUL"]];
```

Comment:
Shouldn't we at least use fileHandleWithNullDevice here? And what about input?

Reply:
I changed it to use the method suggested.  Also, I am not sure input is needed as it is launching gdnc here.   One other question.   Will it matter that the handle is opened for writing if we just use the fileHandleWithNullDevice method?